src/index.js

```diff
@@ -1,6 +1,8 @@
 import React from 'react';
 import { BrowserRouter as Router } from 'react-router-dom';
+import { Provider } from 'react-redux';
 import ReactDOM from 'react-dom/client';
+import store from './redux/configureStore';
 import './index.css';
 import App from './App';
 // import BooksContainer from './components/BooksContainer';
```

Comment:
- [ ] Kindly remove all commented code from your codebase to make your codebase clean and readable 👍🏽

Reply:
Thanks @vickymarz  for your professionalism!